### ISSUE TITLE
having a multiline comment at the end of the file outputs an error

### ISSUE THREAD

**Issue by RecruitMain707** [TiePy/nimatron#7]
![image](https://user-images.githubusercontent.com/37117366/90319590-c9344f80-df39-11ea-8e70-2ef76522c67f.png)

(sorry for bombarding you with bugs, dont feel pressured, you are doing an awesome job and the plugin is going very well)

**Comment by stever** [comment]
Thanks for the reports. I create test cases to help cover these issues. Some not handled by the grammar specification so I think I might need to get more creative with those. I'll dig into the way I can take more control over the parsing at points. I appreciate the testing and patience while I continue to improve this.

**Comment by stever** [comment]
Can you provide a test example? I haven't been able to reproduce this with the current version available.

**Comment by RecruitMain707** [comment]
yes, the error is pretty specific indeed, im gonna test a few things out to see what is actually causing the error to appear sometimes

**Comment by RecruitMain707** [comment]
ok, luckily that didnt take very long, the issue appears because of this `{.pop.}`
![image](https://user-images.githubusercontent.com/37117366/90394092-4a065f00-e092-11ea-9702-e33cff666439.png)

Here it is the whole file, just in case you want to test other things
https://play.nim-lang.org/#ix=2uq8

**Comment by stever** [comment]
Thanks. I'm going to try and figure out a better way of testing the parser. I'm thinking about creating AST output from the parser I've created in IDEA's Grammar-Kit/JFlex and also the Nim parser itself and compare them. Also to work through the Nim parser and see if I can work out some manual parsing rules, if that helps. Comparing the ASTs would be reassuring getting something equivalent and building a better parser. The Nim grammar is complicated and I need to work out where it might be behaving differently to the way it has been specified. I'll just adapt my approach to the problem.
